netbeans/CheckerFrameworkPlugin/nbproject/private/platform-private.properties

```diff
@@ -0,0 +1 @@
+user.properties.file=C:\\Users\\Tony\\AppData\\Roaming\\NetBeans\\8.1\\build.properties
```

Comment:
I'm not sure which of the `nbproject` files should be in the repository. I guess at least the `private` files shouldn't be, as they contain local file paths. Can you look for other netbeans projects on github and see what they store in git?